drivers/media/i2c/imx290.c

```diff
@@ -518,6 +523,26 @@ static int imx290_set_gain(struct imx290 *imx290, u32 value)
 	return ret;
 }
 
+static int imx290_set_hmax(struct imx290 *imx290, u32 val)
+{
+	u32 hmax = val + imx290->current_mode->width;
+	int ret;
+
+	ret = imx290_write_reg(imx290, IMX290_HMAX_LOW, (hmax & 0xff));
```

Comment:
Why are IMX290_HMAX_LOW & _HIGH written explicitly as two 8-bit registers, rather than using imx290_write_buffered_reg(imx290, IMX290_HMAX_LOW, 2, hmax)? Have I missed a subtlety?

Reply:
I'd meant to query this on the mailing list, but missed my chance as they got merged just before I sent my comment and Tested-by: tag.

Reply:
Not that my diff here is only moving the function from below as it needs to be called from imx290_set_ctrl

Reply:
OK - it was just an observation. I initially had a comment against the IMX290_VMAX support before realising that it was correct.

Reply:
I've added an extra patch that converts imx290_set_hmax to use imx290_write_buffered_reg